packages/diagrams/frontend/sirius-components-diagrams/src/palette/tool/Tool.tsx

```diff
@@ -10,36 +10,41 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
-import { ServerContext } from 'common/ServerContext';
-import { Text } from 'core/text/Text';
-import { GenericTool } from 'icons/GenericTool';
-import PropTypes from 'prop-types';
-import React, { useContext } from 'react';
-import styles from './Tool.module.css';
+import { ServerContext } from '@eclipse-sirius/sirius-components-core';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import CropDinIcon from '@material-ui/icons/CropDin';
+import { useContext } from 'react';
+import { ToolProps } from './Tool.types';
 
-const propTypes = {
-  tool: PropTypes.shape({
-    id: PropTypes.string.isRequired,
-    label: PropTypes.string.isRequired,
-    imageURL: PropTypes.string,
-  }).isRequired,
-  selected: PropTypes.bool,
-  onClick: PropTypes.func.isRequired,
-  disabled: PropTypes.bool,
-  thumbnail: PropTypes.bool,
-};
+const useToolStyles = makeStyles(() => ({
+  tool: {
+    display: 'grid',
+    gridTemplateRows: '1fr',
+    gridTemplateColumns: '20px 1fr',
+    alignItems: 'center',
+    cursor: 'pointer',
+  },
+  disabled: {
+    cursor: 'default',
+  },
+  selected: {
+    fontWeight: 'bold',
+  },
+}));
 
-export const Tool = ({ tool, selected, onClick, disabled, thumbnail }) => {
+export const Tool = ({ tool, selected, onClick, disabled, thumbnail }: ToolProps) => {
   const { id, label, imageURL } = tool;
   const { httpOrigin } = useContext(ServerContext);
-  let className = styles.tool;
+
+  const classes = useToolStyles();
+
+  let className = classes.tool;
   if (disabled) {
-    className = `${className} ${styles.disabled}`;
-  } else if (selected) {
-    className = `${className} ${styles.selected}`;
+    className = `${className} ${classes.disabled}`;
   }
 
-  let image = <GenericTool title={label} style={{ fill: 'var(--daintree)' }} />;
+  let image = <CropDinIcon />;
```

Comment:
This changes the UI (from a circle with a lozange inside to a plain rectangle). Is it voluntary?

![t](https://user-images.githubusercontent.com/10608/186380007-d1493529-4c76-4904-8b7f-3fa86c32862a.svg)

Reply:
I thought that this was relying on the svg import so I switched it to a MUI icon but since it does not (I've just checked), I can revert that part. What do you think?

Reply:
Done